platform-hub-web/src/app/kubernetes-tokens/kubernetes-tokens-form.html

```diff
@@ -2,9 +2,10 @@
   <md-toolbar md-scroll-shrink>
     <div class="md-toolbar-tools">
       <h3>
-        <span ng-if="$ctrl.isNew">New Token</span>
+        <span ng-if="$ctrl.isNew && $ctrl.user.is_active">New Token</span>
```

Comment:
I purposely allowed access to deactivated users tokens page so that admin can delete individual tokens. Title at the top will change accordingly by showing "deactivated" badge so this should be good enough?